### PR TITLE
fixit: migrate to new region tag with prefix 'job_'

### PR DESCRIPTION
**Pull request by ymotongpoo**
## Description

- Internal: b/347825844
- Internal: b/347825952
- Internal: b/347825964
- Internal: b/347825934

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed [Contributing Guidelines from CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md)
- [x] **Tests** pass:   `go test -v ./..` (see [Testing](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#testing))
- [x] **Code formatted**:   `gofmt` (see [Formatting](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#formatting))
- [x] **Vetting** pass:   `go vet` (see [Formatting](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#formatting))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Review by iennae — APPROVED** [review]
Thanks for resolving these issues!

**Comment by iennae** [comment]
based on snippet checks, a couple of the snippets need to be a 2-phase where you add the region tag in this PR, submit the CL, wait for doc to get merged, then remove the existing region tag.

File: jobs/v3/howto/location_based_search_sample.go
Comment (iennae) [inline jobs/v3/howto/location_based_search_sample.go:87]: you need to leave city_location_search here until the new region tag has been merged.

File: jobs/v3/howto/location_based_search_sample.go
Comment (iennae) [inline jobs/v3/howto/location_based_search_sample.go:147]: you need to leave broadening_location_search here (both the start and end tags) until the new region tag has been merged in docs.

**Review by iennae — CHANGES_REQUESTED** [review]

File: jobs/v3/howto/location_based_search_sample.go
Comment (ymotongpoo) [reply jobs/v3/howto/location_based_search_sample.go]: Thank you for catching this. I revered this tag for now.

File: jobs/v3/howto/location_based_search_sample.go
Comment (ymotongpoo) [reply jobs/v3/howto/location_based_search_sample.go]: Thank you for catching this. I reverted this tag for now.

Comment (iennae) [reply jobs/v3/howto/location_based_search_sample.go]: what you can do is add the new one, just don't delete the old one and then after CL merges you can delete the old one.

**Review by iennae — APPROVED** [review]

**Comment by ymotongpoo** [comment]
@iennae thank you for your review. given cl/648538021 is submitted, can I delete the corresponding old tags from this PR?

**Pull request merged**
